src/goto-instrument/remove_function.cpp

```diff
@@ -70,3 +70,48 @@ void remove_functions(
   for(const auto &f : names)
     remove_function(goto_model, f, message_handler);
 }
+
+void block_function(
```

Comment:
It would be nice to have some Doxygen comments for this function, as @peterschrammel mentioned.